hazelcast-client/src/main/java/com/hazelcast/client/impl/HazelcastClientManagedContext.java

```diff
@@ -16,17 +16,17 @@
 
 package com.hazelcast.client.impl;
 
-import com.hazelcast.core.HazelcastInstance;
 import com.hazelcast.core.HazelcastInstanceAware;
 import com.hazelcast.core.ManagedContext;
+import com.hazelcast.spi.serialization.SerializationServiceAware;
 
 final class HazelcastClientManagedContext implements ManagedContext {
 
-    private final HazelcastInstance instance;
+    private final HazelcastClientInstanceImpl instance;
```

Comment:
Aligned with `HazelcastManagedContext`, which also uses its `HazelcastInstanceImpl`. The `HazelcastClientManagedContext` is initialized in `DefaultClientExtension` and given the `HazelcastInstanceImpl` as parameter. This also saves a cast in `initialize()` to retrieve the `SerializationService`.